data/stoa0045/stoa001/__cts__.xml

```diff
@@ -1,8 +1,9 @@
 <ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" urn="urn:cts:latinLit:stoa0045.stoa001" xml:lang="lat" groupUrn="urn:cts:latinLit:stoa0045">
             <ti:title xml:lang="lat">Bissula</ti:title>
             <ti:edition urn="urn:cts:latinLit:stoa0045.stoa001.perseus-lat2" workUrn="urn:cts:latinLit:stoa0045.stoa001">
-              <ti:label xml:lang="eng">Bissula, Ausonius, with an English translation Vol 1</ti:label>
-              <ti:description xml:lang="eng">Ausonius, Decimus Magnus, creator; Evelyn-White, Hugh G. (Hugh Gerard), d. 1924, translator, editor; Evelyn-White, Hugh G. (Hugh Gerard), d. 1924, translator</ti:description>
+              <ti:label xml:lang="lat">Bissula</ti:label>
```

Comment:
It looks like the Latin title in the file is **De Bissula** whereas the English title is **Bissula**?

Reply:
Ah I missed that. I will change that and watch for it when I do other Ausonius files.